webpack.prod.config.js

```diff
@@ -13,6 +13,50 @@ function resolvePrivateEnvConfig(filePath) {
 };
 resolvePrivateEnvConfig('.env.private');
 
-const config = createConfig('webpack-prod');
+// Create/extend the Webpack configuration
+const config = createConfig('webpack-prod', {
+  optimization: {
+    moduleIds: 'deterministic',
+    splitChunks: {
+      // `maxSize` applies to all cache groups. Note, per the Webpack documentation, `maxSize` is intended to be used with
+      // HTTP/2 and long term caching, due to increasing the request count for better caching. `maxSize` takes precedence
+      // over `maxInitialRequests` and `maxAsyncRequests`. Note, the 244 KiB value for `maxSize` listed below is based on
+      // parsed size, not Gzipped size.
+      maxSize: 244 * 1024,
+      cacheGroups: {
+        // Disable the default vendors cache group.
+        defaultVendors: false,
+        // Split out the vendor code from the application code.
+        vendors: {
+          test: /[\\/]node_modules[\\/]/,
+          name: 'vendors',
```

Comment:
As discussed asynchronously, since the configuration doesn't allow us to override the name of the package chunk from code splitting to identify which file it maps to. Maybe its worth looking into for greater observability.